amd/src/annoto.js

```diff
@@ -159,28 +160,25 @@ define([
                   domain: params.deploymentDomain
                 },
                 clientId: params.clientId,
-                ux: {
-                    ssoAuthRequestHandle: function() {
-                        window.location.replace(params.loginUrl);
-                    },
-                },
                 hooks: {
                     mediaDetails: function() {
                         return {
                             details: {
                                 title: params.mediaTitle,
                                 id: params.mediaId,
-                                description: params.$cmintro,
+                                description: params.mediaDescription,
                             }
                         };
                     },
+                    ssoAuthRequestHandle: function() {
+                        window.location.replace(params.loginUrl);
+                    },
                 },
                 locale: params.locale,
-                widgets: [{player: {}}],
```

Comment:
why removed? it's needed here.

this comment referred to an entirely different object which is used for wistia, there it's not needed.
![image](https://user-images.githubusercontent.com/6747147/154336818-8e554801-f66c-4cb3-bab6-16928bb439c3.png)